### ISSUE TITLE
Guard and Directive use different logical operator

### ISSUE THREAD

**Issue by zveljkovic** [AlexKhymenko/ngx-permissions#13]
## I'm submitting a...
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
</code></pre>

## Current behavior
```
<ng-template [ngxPermissionsOnly]="['ROLE_ADMIN','ROLE_USER']">
    <div>ROLE_ADMIN | ROLE_USER</div>
</ng-template>
```
View directive is OR-ed  (ROLE_USER || ROLE_ADMIN)

```
{ path: 'quick-quote', component: QuickQuoteStartComponent,
    canActivate: [NgxPermissionsGuard],
    data: { permissions: { only: ['ROLE_USER','ROLE_ADMIN'], redirectTo: loginRedirect} }
  },
```

Route guard is AND-ed (ROLE_USER && ROLE_ADMIN)..

## Expected behavior
They should work the  same 

## Environment

<pre><code>
Angular version: @angular/cli: 1.4.6
ngx-permissions version: 2.1.1
</code></pre>

**Comment by AlexKhymenko** [comment]
Hi. I have checked Route guard uses OR operator as well. Here is plunker.

**Comment by zveljkovic** [comment]
Hi I have investigated more this issue. Here's the plunker

https://plnkr.co/edit/ficIgPfjlQIDiCJ8xIGa?p=preview

Seems that when setting redirectTo to a function then it behaves with AND operator.

**Comment by AlexKhymenko** [comment]
Will investigate asap.

**Comment by AlexKhymenko** [comment]
I have located the issue. Tomorrow or on Thursday will publish new version. Thank You.

**Comment by zveljkovic** [comment]
Thank you!

**Comment by AlexKhymenko** [comment]
After taking fresh look. I have done such behavior cause in such case  

`  permissions: {
               only: ['canReadAgenda','canEditAgenda'],
               redirectTo: {
                 canReadAgenda: 'agendaList',
                 canEditAgenda: 'dashboard',
                 default: 'login'
               }
        }`  It will use and operator. And for me it seems like a valid use case. But i guess i need to make breaking change and change it to or operator. Or add flag and: true so it will use true operator but then i will need to do the same in directive :-)

**Comment by AlexKhymenko** [comment]
The problem is bigger than I thought. Will take more time then i expected.

**Comment by AlexKhymenko** [comment]
Fixed with version 3.0.0
